### ISSUE TITLE
isolation forest model parameter

### ISSUE THREAD

**Issue by Hello-Hyuk** [Hello-Hyuk/Credit_card_fraud#2]
result via params

**defalt param**
n_estimators=125
max_samples=len(train_x)
contamination=val_contamination
random_state=42
verbose=0
warm_start=False

https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.IsolationForest.html

**Comment by Hello-Hyuk** [comment]
**n_estimators=150**
max_samples=len(train_x)
contamination=val_contamination
random_state=42
verbose=0
warm_start=False

**RESULT**
Validation F1 Score : [0.7093608634827273]
              precision    recall  f1-score   support

           0       1.00      1.00      1.00     28432
           1       0.41      0.43      0.42        30

    accuracy                           1.00     28462
   macro avg       0.70      0.72      0.71     28462
weighted avg       1.00      1.00      1.00     28462

**Comment by Hello-Hyuk** [comment]
maybe warm_start has little effect

**Comment by Hello-Hyuk** [comment]
val_x_p = val_df.drop(columns=['ID', 'Class', 'V1', 'V12', 'V20', 'V21', 'V23', 'V27', 'V28', 'V29'])
균등 분포 col + V30

**result**
Validation F1 Score : [0.7330519602326018]
              precision    recall  f1-score   support

           0       1.00      1.00      1.00     28432
           1       0.47      0.47      0.47        30

    accuracy                           1.00     28462
   macro avg       0.73      0.73      0.73     28462
weighted avg       1.00      1.00      1.00     28462

**Comment by Hello-Hyuk** [comment]
**feature**
val_x_p = val_df.drop(columns=['ID', 'Class', 'V1',**'V2','V5','V7','V8',** 'V12', 'V20', 'V21', 'V23', 'V27', 'V28', 'V29'])

**hyper param**
n_estimators=125, 
max_samples=len(train_x_p), contamination=val_contamination, random_state=42,
verbose=0,
warm_start=True)


**result**
Validation F1 Score : [0.7620400831103115]
              precision    recall  f1-score   support

           0       1.00      1.00      1.00     28432
           1       0.52      0.53      0.52        30

    accuracy                           1.00     28462
   macro avg       0.76      0.77      0.76     28462
weighted avg       1.00      1.00      1.00     28462

**Comment by Hello-Hyuk** [comment]
**feature**
val_x_p = val_df.drop(columns=['ID', 'Class', 'V1',**'V2','V5','V7','V8',** 'V12', 'V20', 'V21', 'V23', 'V27', 'V28', 'V29'])

**hyper param**
n_estimators=150, 
max_samples=len(train_x_p), contamination=val_contamination, random_state=42,
verbose=0,
warm_start=True)

**result**
Validation F1 Score : [0.7664204652035265]
              precision    recall  f1-score   support

           0       1.00      1.00      1.00     28432
           1       0.53      0.53      0.53        30

    accuracy                           1.00     28462
   macro avg       0.77      0.77      0.77     28462
weighted avg       1.00      1.00      1.00     28462
